Header header
float64[] vcell
float64 vtotal
float64 current
bool[] batteryState
float64 timestamp
